src/plugins/rehype/rehypeSyntaxHighlighting.ts

```diff
@@ -6,6 +6,10 @@ import type { Node } from 'unist';
 import { Parent } from 'unist';
 import { visit } from 'unist-util-visit';
 
+import blade from '../../lib/syntaxHighlighting/blade.js';
+
+refractor.register(blade);
```

Comment:
Is this just cause it's an extension?

Reply:
yea, we need to register to so that refactor can recognize `blade` is one of the registered language